README.md

```diff
@@ -1,137 +1,142 @@
-
-This project is [MIT](./LICENSE) licensed.
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+  -[📄 Presentation](#presentation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 coinRate
+ <a name="about-project"></a>
+
+> A mobile web application that retrives latest price of popular cryptocurrencies from an API
+
+**coinRate** displays a list of crypto coins form API!
+
+## 🛠 Built With <a name="built-with">React</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://coinrate.onrender.com)
+
+## 📄 Presentation <a name="presentation"></a>
+
+- [Loom Link](https://www.loom.com/share/a4aa2958465c41e89d3fd267ba7638b1)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Node.js
+- Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+```sh
+  -cd my-folder
+  -git clone git@github.com:mrsamsharma/coinRate.git
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd coinRate
+
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+`-npm start`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Satyam Sharma**
+
+- GitHub: [@samWfelcie](https://github.com/mrsamsharma)
+- Twitter: [@samwfelice](https://twitter.com/samwfelice)
+- LinkedIn: [samfelice](https://www.linkedin.com/in/samfelice)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Will diplay crypto coins from API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can star it :)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Nelson Sakwa for the design of this app

```

Comment:
- [ ] [optional] It would have been nice if you added a link to his  Behance page